nuxt-frontend/pages/about.vue

```diff
@@ -14,81 +14,80 @@
+const HowItAllStartedSectionMobile = defineAsyncComponent(
+  () => import("@/components/about/HowItAllStartedSectionMobile.vue"),
 );
-const AboutusVirtue = defineAsyncComponent(() =>
-  import("@/components/about/AboutusVirtue.vue"),
+const AboutusVirtue = defineAsyncComponent(
+  () => import("@/components/about/AboutusVirtue.vue"),
 );
-const WithCanopasSection = defineAsyncComponent(() =>
-  import("@/components/about/WithCanopas.vue"),
+const WithCanopasSection = defineAsyncComponent(
+  () => import("@/components/about/WithCanopas.vue"),
 );
-const ClientReviewSection = defineAsyncComponent(() =>
-  import("@/components/home-new/ClientReviewSection.vue"),
+const ClientReviewSection = defineAsyncComponent(
+  () => import("@/components/home-new/ClientReviewSection.vue"),
 );
-const CTASection = defineAsyncComponent(() =>
-  import("@/components/about/CTASection.vue"),
+const CTASection = defineAsyncComponent(
+  () => import("@/components/about/CTASection.vue"),
 );
-const NewFooter = defineAsyncComponent(() =>
-  import("@/components/partials/NewFooter.vue"),
+const NewFooter = defineAsyncComponent(
+  () => import("@/components/partials/NewFooter.vue"),
 );
+const { $mixpanel } = useNuxtApp();
 
-export default {
-  setup() {
-    const seoData = config.ABOUT_SEO_META_DATA;
-    useSeoMeta({
-      title: seoData.title,
-      description: seoData.description,
-      ogTitle: seoData.title,
-      ogType: seoData.type,
-      ogUrl: seoData.url,
-      ogImage: seoData.image,
-    });
-  },
-  components: {
-    Header,
-    LandingSection,
-    HowItAllStartedSectionMobile,
-    HowItAllStartedSection,
-    AboutusVirtue,
-    WithCanopasSection,
-    ClientReviewSection,
-    CTASection,
-    NewFooter,
-  },
-  data() {
-    return {
-      event: "",
-      events: {
-        withcanopas: "view_about_phases",
-        footer: "view_about_footer",
-      },
-    };
-  },
-  inject: ["mixpanel"],
-  mounted() {
-    window.addEventListener("scroll", this.sendEvent);
-    this.$mixpanel.track("view_about_page");
-  },
-  unmounted() {
-    window.removeEventListener("scroll", this.sendEvent);
-  },
-  methods: {
-    sendEvent() {
-      const event = this.events[elementInViewPort(this.$refs)];
-      if (event && this.event !== event) {
-        this.event = event;
-        this.$mixpanel.track(event);
-      }
-    },
-  },
+const landingsection = ref(null);
+const howitallstartedsectionmobile = ref(null);
+const howitallstartedsection = ref(null);
+const aboutusvirtue = ref(null);
+const withcanopassection = ref(null);
+const clientreviewsection = ref(null);
+const ctasection = ref(null);
+
+const seoData = config.ABOUT_SEO_META_DATA;
+useSeoMeta({
+  title: seoData.title,
+  description: seoData.description,
+  ogTitle: seoData.title,
+  ogType: seoData.type,
+  ogUrl: seoData.url,
+  ogImage: seoData.image,
+});
+
+let event = "";
+let events = {
+  landing: "view_about_landing",
```

Comment:
As we are sending the mixpanel event on page load,  `landing` section event is not required